lib/qmp.ml

```diff
@@ -47,16 +47,38 @@ type qom = {
   ty   : string;
 }
 
-type params = {
-  bus     : string;
-  hostbus : string;
-  hostport: string;
-}
-
-type device = {
-  driver : string;
-  id     : string;
-  params : params option;
+module Device = struct
+  module USB = struct
+    module Driver = struct
+      type t = USB_EHCI | USB_HOST
+      let string_of = function
```

Comment:
Isn't this more conventionally called `to_string`?

Reply:
I find more symmetric the convention of_string and string_of, as I see in Pervasives. But I can see that other libraries use to_string -- if you prefer, I can easily change that.

Reply:
I was wrong in my impression that there is a universal convention - it is not and so just keep it. We have `string_of_int` and `int_of_string` but also `Printexc.to_string`.